portfolio/src/main/webapp/funfacts.html

```diff
@@ -29,6 +30,38 @@ <h2>Click here to generate a random fun fact about me:</h2>
             <img src="images/xkcd_pointers.png" id="xkcd-pointers" alt="XKCD Pointers Cartoon" width="300">
             <p>Here's a fun cartoon about pointers for the C/C++ enthusiasts reading my website.</p>
         </div>
+        <div class="boxed-div">
```

Comment:
You are repeating code in funfacts.html and hobbies.html. Is there a way generate the html in js. We can pass in an argument for the location: Funfacts and Hobbies

The reason we don't want repeated code is that its hard to maintain. If you make a change in funfacts, you also have to make the change in hobbies.

Reply:
**Note: ** All of the other pull review comments should be addressed in my newest commits from June 8th (today). This one has not been addressed yet, and I will work on it later in the day. I just wanted to write this to clarify what I did / did not fix yet.

Reply:
Hi Cassandra, im going to approve the PR. If this will take a long time, you don't need to worry about it. But if you do make the changes, I'll look at the change.